src/stat.cc

```diff
@@ -1917,10 +1917,3 @@ statGraphDump(StoreEntry * e)
 }
 
 #endif /* STAT_GRAPHS */
```

Comment:
PR removes a customary empty line from the end of this file. If possible, please format changed sources before merging.

```suggestion
#endif /* STAT_GRAPHS */

```